pkg/gitparse/gitparse.go

```diff
@@ -152,14 +189,13 @@ func (c1 *Commit) Equal(c2 *Commit) bool {
 			return false
 		case d1.LineStart != d2.LineStart:
 			return false
-		case d1.Content.String() != d2.Content.String():
+		case d1.contentWriter.String() != d2.contentWriter.String():
```

Comment:
doesn't `bufferedFileWriter.String()` only stringify the in-memory buffer? i might have missed something.

Reply:
err whoops, it should be the whole diff contents. Will add a test and update.